examples/webrtc/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <body>
+    <h3>Browser base64 Session Description</h3>
+    <textarea id="localSessionDescription" readonly="true"></textarea>
+
+    <h3>Golang base64 Session Description</h3>
+    <textarea id="remoteSessionDescription"></textarea>
+    <button onclick="window.startSession()">Start Session</button>
+
+    <h3>Video</h3>
+    <div id="remoteVideos"></div>
+
+    <h3>Logs</h3>
+    <div id="logs"></div>
+
+
+</head>
+<meta charset="utf-8">
+
+<script>
+  window.addEventListener("load", function(evt) {
+        /* eslint-env browser */
+
+        const pc = new RTCPeerConnection({
+          iceServers: [{
+            urls: 'stun:stun.l.google.com:19302'
+          }]
+        })
+        const log = msg => {
+          document.getElementById('logs').innerHTML += msg + '<br>'
+        }
+
+        pc.ontrack = event => {
+          const el = document.createElement(event.track.kind)
+          el.srcObject = event.streams[0]
+          el.autoplay = true
+          el.controls = true
+          document.getElementById('remoteVideos').appendChild(el)
+        }
+
+        pc.onsignalingstatechange = e => log(pc.signalingState)
+        pc.oniceconnectionstatechange = e => log(pc.iceConnectionState)
+        pc.onicecandidate = event => {
+          if (event.candidate === null) {
+            document.getElementById('localSessionDescription').value = btoa(JSON.stringify(pc.localDescription))
+          }
+        }
+
+        // Offer to receive 1 audio, and 1 video tracks
+        pc.addTransceiver('audio', {
+          'direction': 'recvonly'
+        })
+        pc.addTransceiver('video', {
+          'direction': 'recvonly'
+        })
+
+
+        window.startSession = () => {
+          const sd = document.getElementById('remoteSessionDescription').value
+          if (sd === '') {
+            return alert('Session Description must not be empty')
+          }
+          try {
+            pc.setRemoteDescription(new RTCSessionDescription(JSON.parse(atob(sd)))).catch(log)

```

Comment:
Here I just copied how the data-channels-create is set up. Golang provides an offer, the browser gives an answer. After ` pc.createAnswer().then(d => pc.setLocalDescription(d)).catch(log)` is run, the browser starts gathering ice candidates and the final candidate gets pasted in the browser base64 description (line 47).